draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1820,44 +1820,41 @@ IDNA2003 {{RFC3490}}.
 
 # Privacy Considerations
 
-Cookies are often criticized for letting servers track users. For example, a
-number of "web analytics" companies use cookies to recognize when a user returns
-to a web site or visits another web site. Although cookies are not the only
-mechanism servers can use to track users across HTTP requests, cookies
-facilitate tracking because they are persistent across user agent sessions and
-can be shared between hosts.
+The primary privacy risk associated with Cookies is their ability to correlate user activity. This can happen on a single site, but is most problematic when activity is tracked across different, seemingly unconnected Web sites to build a profile of the user.
+
+Over time, this capability (warned against explicitly in {{?RFC2109}} and all of its successors) has become widely used to authenticate users across sites, to combat fraud and other forms of undesirable traffic, to target advertisements at specific users or at users with specified attributes, to measure how often ads are shown to users, and to recognise when an ad resulted in a change in user behavior.
```

Comment:
```suggestion
Over time, this capability (warned against explicitly in {{?RFC2109}} and all of its successors) has become widely used to correlate user activity across sites, known also as tracking. Reasons are varied and include protection against fraud and other forms of undesirable traffic, assembling information on users, targeting advertisements at specific users or at users with specified attributes, measuring how often ads are shown to users, and to recognize when an ad resulted in a change in user behavior.
```

Reply:
committing, but it needs some tweaks and you're losing federated auth...